.docker/scripts/init-ext.sh

```diff
@@ -4,13 +4,12 @@
 #
 set -e
 
-. /app/ckan/default/bin/activate
-
-pip install -r "/app/requirements.txt"
-pip install -r "/app/requirements-dev.txt"
-python setup.py develop
+PIP="${APP_DIR}/bin/pip"
```

Comment:
would this have issues with python3 2.9 or will we cross that at a later date

Reply:
I was leaving that for later.